### ISSUE TITLE
Exception on creating a WpfPlot object

### ISSUE THREAD

**Issue by MatthiasMuellerDev** [ScottPlot/ScottPlot#4255]
Hello,

I am currently programming a software for evluation of large amounts of data. The software will be part of a greater project and is based on the Microsoft .Net Framework Version 4.6.2.

All tests on my development PC are working fine. But after I installed the software at the target computer I am getting an exception while creating a WpfPlot object:

> The type initializer for “ScottPlot.Fonts” has caused an exception.
Callstack:
> at ScottPlot.LabelStyle..ctor()
> at ScottPlot.Plottables.Benchmark..ctor()
> at ScottPlot.Plot..ctor()
> at ScottPlot.WPF.WpfPlotBase..ctor()

It seems, that the exception object contains an inner exception with the following callstack:
> at SkiaSharp.SKTypeface.get_Default()
> at ScottPlot.FontResolvers.SystemFontResolver.InstalledSansFont()
> at ScottPlot.Fonts..cctor() 

I have no idea what is the reason for the exception or what to do to fix the problem. Any Help is appreciated.

Because of the call stack of the inner exception I suppose, it is something around fonts. I found, that on my development PC the font "Segoe UI" is used. So I checked the font at the target PC. On the target PC the font is installed in a different version to my development PC: On my development PC "Standard Segoe UI" is installed in Version 5.62 and on the target PC in Version 5.53. Because it is a system font it is not possible to update the font. Can this be the reason of the problem or it is random difference?

|  | target PC | development PC |
| ------------------ | ---------------------------------- | ------------------------ |
| operating system | Windows 10 Enterprise 2016 LTSB Version 1607 | Windows 10 Pro 22H2 |
| installed RAM | 8 GB | 16 GB |
| system type | 64-Bit-operating system, x64-based processor | 64-Bit-operating system, x64-based processor |
| .Net | .Net Framework 4.6.2 | .Net Framework 4.6.2 |

I am using the version 5.0.39.

Many thanks in advance for any help.

Matthias Müller

**Comment by kebox7** [comment]
@MatthiasMuellerDev,  check out these links, they discuss similar issues. Maybe this will help you find a solution.

[Error occurs only after installing application: The type initiliazer for 'SkiaSharp.SKTypeface' threw an exception](4157)
[libSkiaSharp missing error - Azure function (windows)](4129)
[System.Runtime.InteropServices.RuntimeInformation issue on .NET Framework](3911)
[System.DllNotFoundException: Unable to load library 'libSkiaSharp'](3848)
[[Solved] ScottPlot 5 Linux TypeFace issues due to missing DLL](3357)

**Comment by MatthiasMuellerDev** [comment]
Hello,

Thank you @kebox7 for the links to the other issues. You are right with your suspicion that it is a problem of SkiaSharp library.

After I had the chance to take a look at the message of the inner exception, it was clear to me that the library libSkiaSharp.dll was missing. The reason for the exception is quite simple. I did not copy the file to the target computer, because it was new to me that in the output directory can exists architecture depend files. So I did not notice the new directory "x64" in the output directory.

Thank you!
